### PR TITLE
AACT-415: Add an export for the Query

### PR DESCRIPTION
**Pull request by Javier-Jimenez-18**
- Modified the `QueryController` index action so that it responds to the csv format.
- Created an index.csv.erb view file. 
- Modified Query index action in index.html.erb view file. Added a button/link to export the data from query.csv file. 
- Updated the `/query` route ---> `query_path`.

Screen Shot 2022-08-22 at 5 44 42 PM
Screen Shot 2022-08-22 at 5 45 18 PM
Screen Shot 2022-08-22 at 5 45 37 PM
Screen Shot 2022-08-22 at 7 01 37 PM

### CODE REVIEW

**Comment by Javier-Jimenez-18** [comment]
- Generated the csv file in ruby by modifying the QueryController index action.
- Deleted the index.csv.erb view file.

Screen Shot 2022-08-23 at 3 32 16 PM
Screen Shot 2022-08-23 at 3 33 34 PM
Screen Shot 2022-08-23 at 3 34 21 PM
Screen Shot 2022-08-23 at 3 34 55 PM
Screen Shot 2022-08-23 at 3 39 25 PM

**Comment by Javier-Jimenez-18** [comment]
Updated and added more coverage on the controller RSpec test file:

Screen Shot 2022-08-26 at 4 52 24 PM

Updated and added more coverage on the requests RSpec test file:

Screen Shot 2022-08-26 at 4 53 36 PM

File: app/views/query/index.html.erb [hunk 1]
```diff
@@ -22,7 +22,9 @@
 <% if @results %>
   <div class="container">
     <div class="pb-2">
-      <h1>SQL Query Results</h1>
+      <h1>SQL Query Results &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;
```
Comment (micronix) [inline app/views/query/index.html.erb:25]: use bootstrap instead of multiple spaces

**Review by micronix — CHANGES_REQUESTED** [review]

File: app/views/query/index.html.erb (hunk 1 above)
Comment (Javier-Jimenez-18) [reply app/views/query/index.html.erb:25]: I used the bootstrap float utility class and removed the HTML non breaking space tabs.

**Comment by Javier-Jimenez-18** [comment]
I used the bootstrap float utility class for the Export Query CSV button:

Screen Shot 2022-09-01 at 8 49 07 AM

**Review by micronix — APPROVED** [review]
Looks great!

**Pull request merged**